django_app/redbox_app/settings.py

```diff
@@ -323,3 +323,20 @@
 USE_STREAMING = env.bool("USE_STREAMING")
 FILE_EXPIRY_IN_SECONDS = env.int("FILE_EXPIRY_IN_DAYS") * 24 * 60 * 60
 SUPERUSER_EMAIL = env.str("SUPERUSER_EMAIL", None)
+
+# Security classifications
+# https://www.gov.uk/government/publications/government-security-classifications/
+
+MAX_SECURITY_CLASSIFICATION = env.str("MAX_SECURITY_CLASSIFICATION")
```

Comment:
We could make this an enum:

```python
class Classification(enum.StrEnum):
    OFFICIAL = "Official"
    OFFICIAL_SENSITIVE = "Official Sensitive"
    ...
    
MAX_SECURITY_CLASSIFICATION = Classification[env.str("MAX_SECURITY_CLASSIFICATION")]
```

We'd need to use underscores rather than dashes in the names for them, of course.

Reply:
Also - would we want a default value?

Reply:
I'm fine withw/without an enum. r.e. default, we want it to fall over if they haven't set it